tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -323,6 +336,54 @@ def test_llmobs_chain_schema_io(self, langchain, mock_llmobs_span_writer, mock_t
         )
         _assert_expected_llmobs_llm_span(trace[1], mock_llmobs_span_writer, mock_io=True)
 
+    @pytest.mark.skipif(sys.version_info < (3, 10, 0), reason="Requires unnecessary cassette file for Python 3.9")
+    def test_llmobs_embedding_query(self, langchain, mock_llmobs_span_writer, mock_tracer):
+        embedding_model = langchain.OpenAIEmbeddings()
+        span = self._embed_query(
+            embedding_model=embedding_model,
+            query="hello world",
+            mock_tracer=mock_tracer,
+            cassette_name="openai_embedding_query.yaml",
+        )
+        assert mock_llmobs_span_writer.enqueue.call_count == 1
+        mock_llmobs_span_writer.enqueue.assert_called_with(
+            _expected_llmobs_llm_span_event(
+                span,
+                span_kind="embedding",
+                model_name=embedding_model.model,
+                model_provider="openai",
+                input_documents="hello world",
```

Comment:
`input_documents` should be a list of dictionaries containing the `text` field, hopefully [this](https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/llmobs/utils.py#L47) may give you more context (we only use text field for embeddings documents, ignore the other fields)